src/commands.rs

```diff
@@ -31,9 +31,16 @@ pub struct CommonOptions {
 }
 
 impl CommonOptions {
+    /// Locks the client storage.
+    pub fn lock_storage(&self) -> Result<FileSystemStorage> {
+        FileSystemStorage::lock(&self.storage, |path| {
```

Comment:
As discussed, the point was as a cheap mechanism to allow library users to print a message just prior to the blocking operation.

This will be changed to expose a `try_lock` implementation that can be called first, allowing consumers to print a message if desired (or just bail the operation), and then proceed to call `lock`.